pkg/kubelet/cm/topologymanager/topology_manager.go

```diff
@@ -135,6 +135,11 @@ var _ Manager = &manager{}
 func NewManager(topology []cadvisorapi.Node, topologyPolicyName string, topologyScopeName string, topologyPolicyOptions map[string]string) (Manager, error) {
 	klog.InfoS("Creating topology manager with policy per scope", "topologyPolicyName", topologyPolicyName, "topologyScopeName", topologyScopeName)
 
+	if topologyPolicyName == PolicyNone {
+		policy := NewNonePolicy()
```

Comment:
remove this and just initiate it in `NewNoneScope`

Reply:
perhaps add a one-line comment emphasizing we that with policy=none the scope is not relevant, so we can short circuit here